src/hardware/siglent-sds/api.c

```diff
@@ -106,7 +106,7 @@ static const uint64_t timebases[][2] = {
 
 static const uint64_t vdivs[][2] = {
 	/* microvolts */
-	{ 500, 100000 },
+	{ 500, 1000000 },
```

Comment:
Maybe we'd want to split this to another bugfix PR, although it depends on the maintainers.